feathr-impl/src/main/scala/com/linkedin/feathr/offline/job/LocalFeatureJoinJob.scala

```diff
@@ -52,7 +52,7 @@ object LocalFeatureJoinJob {
       outputPath)
 
     val jobContext = FeatureJoinJob.parseInputArgument(defaultParams ++ extraParams).jobJoinContext
-    SparkFeaturizedDataset(feathrClient.joinFeatures(joinConfig, observationData, jobContext).data, FeaturizedDatasetMetadata())
+    SparkFeaturizedDataset(feathrClient.joinFeaturesWithSwallowedExceptions(joinConfig, observationData, jobContext)._1.data, FeaturizedDatasetMetadata())
```

Comment:
Could we add a comment here describing the context of why we are accessing a specific index.